app/components/pages/Dashboard/ManuscriptsQuery.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { Query } from 'react-apollo'
+
+const ManuscriptsQuery = ({ query, children }) => (
+  <Query fetchPolicy="cache-and-network" query={query}>
+    {({ data, loading, error }) => {
+      if (loading && !data.manuscripts) {
+        return <div>Loading...</div>
+      }
+
+      if (error) {
+        return <div>{error.message}</div>
```

Comment:
At some point we might want to design a view for this, @chugginselifesciences ?

Reply:
I agree, I think for the current batch of work this should be fine as the loading message shouldn't be noticeably visible as long as you're on a half decent connection but It would be good to have a styled message or 'spinner' in the future.

Reply:
Yeah I think the loading screen would be ideal, but the lack of error screen could conceivably be more important. Not only for slow internet connections, which is a very real possibility, but what if there is an error on our side?